src/providers/ses.rs

```diff
@@ -50,7 +50,7 @@ impl Provider for SesProvider {
     ) -> Result<String, ProviderError> {
         let mut destination = Destination::default();
         destination.to_addresses = Some(vec![to.to_string()]);
-        if cc.len() > 0 {
+        if !cc.is_empty() {
```

Comment:
...like this one for instance. Is it objectively clearer? I'm not sure. Normally I prefer to avoid `!` conditions because they're so easy to misread when you're scanning code quickly.

Reply:
(in the other FxA repos we mitigate that to some extent by adding whitespace after the `!`, but unfortunately rustfmt complains when we try to do that here)